tests/CMakeLists.txt

```diff
@@ -24,12 +24,6 @@ target_link_libraries(test_distributedcombigrid_boost PUBLIC discotec Boost::boo
 target_include_directories(test_distributedcombigrid_boost PUBLIC Boost::headers)
 set_property(TARGET test_distributedcombigrid_boost PROPERTY CROSSCOMPILING_EMULATOR '${MPIEXEC_EXECUTABLE} ${MPIEXEC_NUMPROC_FLAG} 9')
 
-if(DISCOTEC_USE_HIGHFIVE)
-    find_package(HighFive REQUIRED)
-    target_link_libraries(test_distributedcombigrid_boost PUBLIC HighFive)
-    target_compile_definitions(test_distributedcombigrid_boost PUBLIC HAVE_HIGHFIVE)
```

Comment:
I am not sure if these should all go; at least the HAVE_HIGHFIVE flag should be passed.